sycl/test/basic_tests/unused_pointer.cpp

```diff
@@ -0,0 +1,43 @@
+// RUN: %clangxx -fsycl -fsycl-targets=%sycl_triple %s -o %t.out
+// RUN: env SYCL_DEVICE_TYPE=HOST %t.out
+// RUN: %CPU_RUN_PLACEHOLDER %t.out
+// RUN: %GPU_RUN_PLACEHOLDER %t.out
+// RUN: %ACC_RUN_PLACEHOLDER %t.out
+
+//==---------- unused_pointer.cpp - test pointers in struct --------------==//
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//==----------------------------------------------------------------------==//
```

Comment:
I think this part isn't needed here. See other LIT tests for reference.

Reply:
Thanks for the review @s-kanaev ! The other tests I checked in in basic_tests had the license information, which is why I added this. Should I remove this?

Reply:
I'm OK with that as long it's aligned with the other tests.

Reply:
Ok. Thank you